util.py

```diff
@@ -208,6 +222,8 @@ def get_z_score(key, value):
     score = 0
     if key == 'z' or key == 'height' or key == 'depth' or key == 'altitude':
         score += 1000
+    if key == 'alt':
+        socre += 100
```

Comment:
`score` instead of `socre`

Reply:
This is why this library needs unit tests.